### PR TITLE
have notplayer target better with more then 2 players

### PR DESCRIPTION
**Pull request by bgkillas**
i couldn't find how you determined when wall is in the way so here is the basic implementation

### CODE REVIEW

**Comment by bgkillas** [comment]
this still needs to fix notplayers shooting each other

**Comment by bgkillas** [comment]
and targeting enemies behind walls

**Comment by bgkillas** [comment]
i do not get why EntityGetName(entity)~="notplayer" does not work to find if the entity is notplayer

**Comment by bgkillas** [comment]
he should stun and do more damage on kick, idk how to do that

File: quant.ew/files/system/notplayer_ai/notplayer_ai.lua [hunk 1]
```diff
@@ -119,20 +116,41 @@ local function update()
         target = nil
     end
 
-    if target == nil then
+--    if target == nil or (GameGetFrameNum() % 4 == 0) then
```
Comment (elmage78) [inline quant.ew/files/system/notplayer_ai/notplayer_ai.lua:119]: Why is this line commented ? may i ask

File: quant.ew/files/system/notplayer_ai/notplayer_ai.lua [hunk 2]
```diff
@@ -68,10 +68,7 @@ end
 local target = nil
 
 local function is_suitable_target(entity)
-    if not EntityGetIsAlive(entity) then
-        return false
-    end
-    return true
+    return EntityGetIsAlive(entity) and EntityGetName(entity)~="notplayer"
```
Comment (elmage78) [inline quant.ew/files/system/notplayer_ai/notplayer_ai.lua:71]: does this work ? `EntityGetName(entity)~="notplayer"`

**Comment by bgkillas** [comment]
@elmage78 this was 40 days ago, many things have changed, those 2 things you pointed out where incorrect and i dont remember why i commented it out

**Pull request merged**